dev/build/windows/makecoq_mingw.sh

```diff
@@ -1164,6 +1164,10 @@ function make_coq {
         COQ_BUILD_PATH=/build/coq-local
         tar -zcf $TARBALLS/coq-local.tar.gz --exclude-vcs -C "${COQ_VERSION%/*}" "${COQ_VERSION##*/}"
```

Comment:
Looking at the `tar` manpage there is
~~~
       --exclude-vcs-ignores
              Exclude files that match patterns read  from  VCS-specific  ignore  files.   Supported
              files are: .cvsignore, .gitignore, .bzrignore, and .hgignore.
~~~
Does this do what you want?